src/Resources/RNGManager.h

```diff
@@ -0,0 +1,34 @@
+/*****************************************************************************
+ * Copyright (c) 2018-2022 openblack developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/openblack/openblack
+ *
+ * openblack is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include <atomic>
+#include <mutex>
+#include <random>
+
+namespace openblack::resources
+{
+class RNGManager
+{
+public:
+	static RNGManager& GetInstance();
+	int NextInt(int min = 1, int max = 500);
```

Comment:
I think uint32 would be better and default value agent really useful for ints

Reply:
Should i keep allowing to define bounds on call of NextInt like in 7fca844?